src/Bounds.cpp

```diff
@@ -142,20 +142,33 @@ class Bounds : public IRVisitor {
         return do_log;
     }
 
+public:
     int interval_log_indent = 0;
 
-    void log_interval(const std::string &msg) const {
+private:
+    std::string log_interval_spaces() const {
+        return std::string(interval_log_indent * 2, ' ');
+    }
+
+    void log_interval_exit(const std::string &msg) const {
```

Comment:
Is `msg` being used in this method?

Reply:
Not at present. (It was before). Left in because it might be useful for future augmentation.